test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Windows/WindowsSnapshotProviderTests.cs

```diff
@@ -155,4 +157,15 @@ public void Provider_Returns_MemoryConsumption()
         var usage = WindowsSnapshotProvider.GetMemoryUsageInBytes();
         Assert.InRange(usage, 0, long.MaxValue);
     }
+
+    [Fact]
+    public void Provider_Creates_Meter_With_Correct_Name()
+    {
+        using var meterFactory = new TestMeterFactory();
+
+        _ = new WindowsSnapshotProvider(_fakeLogger, meterFactory, _options);
+
+        var meter = meterFactory.Meters.Single();
+        Assert.Equal(ResourceUtilizationInstruments.MeterName, meter.Name);
```

Comment:
If we change the value of this constant, will the test pass? I think it will, but it should fail because our goal is to prevent that constant to be changed. What do you think?

Reply:
If you want to break something in the code, you will always be able to do so. But changing a constant value usually requires more consideration and thinking about all the consequences.
If we want to rely on the namespace (which won't change), we would have to use reflection to get to it (`typeof`). Wouldn't it break possible use with Native AOT?